src/main/scala/codacy/duplication/pmd/Cpd.scala

```diff
@@ -25,10 +26,21 @@ object Cpd extends IDuplicationImpl {
 
     val res = Try(cpd.getMatches.toList.map(matchToClone(_, path)))
 
-    System.setOut(outStream)
-    System.setErr(errStream)
+    System.setOut(Console.out)
+    System.setErr(Console.err)
 
-    res
+    res match {
```

Comment:
I favour using ".recoverWith" in these cases, but not mandatory.

Reply:
Try has a recoverWith?